### ISSUE TITLE
Cannot import name 'dimensions' from 'aicsimageio'

### ISSUE THREAD

**Issue by iezqrom** [bkntr/napari-brainways#29]
* Brainways version: 0.1.14
* Python version: 3.9, 3.10 * 3.12
* Operating System: Ubuntu 22.04.5 LTS 

### Description

**Describe what you were trying to get done.**
I was trying to install and run brainways in my computer as I've done in the past. 

**Tell us what happened, what went wrong, and what you expected to happen.**
I get an error with aicsimageio when I run brainways ui in 3.9, 3.10 and 3.12

### What I Did

 brainways ui
```
Traceback (most recent call last):
  File "/home/iezquer/miniforge3/envs/brainways/bin/brainways", line 5, in <module>
    from brainways.scripts.cli import cli
  File "/home/iezquer/miniforge3/envs/brainways/lib/python3.12/site-packages/brainways/scripts/cli.py", line 4, in <module>
    from brainways.scripts.cell_detection import cell_detection
  File "/home/iezquer/miniforge3/envs/brainways/lib/python3.12/site-packages/brainways/scripts/cell_detection.py", line 6, in <module>
    from brainways.project.brainways_project import BrainwaysProject
  File "/home/iezquer/miniforge3/envs/brainways/lib/python3.12/site-packages/brainways/project/brainways_project.py", line 15, in <module>
    from brainways.pipeline.brainways_pipeline import BrainwaysPipeline
  File "/home/iezquer/miniforge3/envs/brainways/lib/python3.12/site-packages/brainways/pipeline/brainways_pipeline.py", line 14, in <module>
    from brainways.project.info_classes import SliceInfo
  File "/home/iezquer/miniforge3/envs/brainways/lib/python3.12/site-packages/brainways/project/info_classes.py", line 12, in <module>
    from brainways.utils.io_utils.readers import QupathReader
  File "/home/iezquer/miniforge3/envs/brainways/lib/python3.12/site-packages/brainways/utils/io_utils/readers/__init__.py", line 6, in <module>
    from brainways.utils.io_utils.readers.qupath_reader import QupathReader
  File "/home/iezquer/miniforge3/envs/brainways/lib/python3.12/site-packages/brainways/utils/io_utils/readers/qupath_reader.py", line 12, in <module>
    from aicsimageio import dimensions, exceptions, transforms
ImportError: cannot import name 'dimensions' from 'aicsimageio' (/home/iezquer/miniforge3/envs/brainways/lib/python3.12/site-packages/aicsimageio/__init__.py)```

**Comment by iezqrom** [comment]
the issue is only with the latest version but not with 1.13.1

**Comment by bkntr** [comment]
Thank you for reporting this issue Ivan. I'll be away until 18th October
and will look into it once I am back.

**Comment by nfournier80** [comment]
Received the same error, working with Mac AppleM3 max and terminal

**Comment by bkntr** [comment]
Should be fixed in v0.1.14.2, sorry for the late response.
